src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -162,7 +164,8 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.custom_command('renew-key', 'iot_hub_policy_key_renew', supports_no_wait=True)
 
     # iot hub routing endpoint commands
-    with self.command_group('iot hub routing-endpoint', client_factory=iot_hub_service_factory) as g:
+    with self.command_group('iot hub routing-endpoint', client_factory=iot_hub_service_factory,
+                            deprecate_info=self.deprecate(redirect=ENDPOINT_DEPRECATION_INFO)) as g:
```

Comment:
```suggestion
                            deprecate_info=self.deprecate(redirect=ENDPOINT_DEPRECATION_INFO, hide=True)) as g:
```

Reply:
I see no issues - will hide both

Reply:
Added the hide to both and also a command that we had marked as deprecated for a while